webapp/src/core.ts

```diff
@@ -11,6 +11,8 @@ import * as pkg from "./package";
 import Cloud = pxt.Cloud;
 import Util = pxt.Util;
 
+export let highContrast: boolean;
```

Comment:
core.ts doesn't have access to the state, and msg alerts show up at the top level (above content and root), and so the standard trick of applying the hc class on root won't work for them, hence why we need an extra field to communicate that across.